### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

**Comment by MikeMcC399** [comment]
"COVID 19" should be written as "COVID-19" (with a hyphen).

Refer to:

- https://www.coronawarn.app/en/
- https://www.zusammengegencorona.de/en/basic-knowledge-about-coronavirus/

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

File: resources/i18n/en/strings.xml [hunk 1]
```diff
@@ -121,4 +116,18 @@
   <!-- XTXT: DCC Reissuance for extension - Consent - Subtitle Text -->
   <string name="DCC_REISSUANCE_CONSENT_SUBTITLE_TEXT_EXTEND">"Request renewal directly in the app"</string>
 
+  <!-- XHED: Mask State - MASK_REQUIRED Card Title -->
+  <string name="MASK_STATE_MASK_REQUIRED_TITLE_TEXT">"Mask Mandate"</string>
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Body - Federal default rules -->
+  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT_BASIC">"You are not exempt from the requirement to wear a mask.\n\nTo be exempt from the mask mandate, you must have had a negative diagnosis in the last 24 hours."</string>
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Body - Extended federal state-specific rules -->
+  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT_EXTENDED">"You are not exempt from the requirement to wear a mask.\n\nTo be exempt from the mask mandate, you must have had a vaccination or recovered from a COVID 19 infection within the last 90 days or had a negative diagnosis in the last 24 hours."</string>
```
Comment (MikeMcC399) [inline resources/i18n/en/strings.xml:124]: Should be COVID-19

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There is still an instance of COVID 19 (without hyphen) in the English language file.

**Comment by mlenkeit** [comment]
@MikeMcC399 thank you again for the thorough review. With https://github.com/corona-warn-app/cwa-app-ccl/pull/112/commits/4c0fb856125e034a599944d5b05a9b77811e6233, all occurrences should be fixed 🚀

**Review by MikeMcC399 — APPROVED** [review]
The instances of "COVID 19" have all been replaced by the correct rendering "COVID-19".

**Pull request merged**